internal/cmd/envs.go

```diff
@@ -169,27 +164,61 @@ coder envs create --image 5f443b16-30652892427b955601330fa5 my-env-name
 coder envs create --cpu 4 --disk 100 --memory 8 --image 5f443b16-30652892427b955601330fa5 my-env-name`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if img == "" {
-				return xerrors.New("image id unset")
+				return xerrors.New("image unset")
 			}
+
+			multiOrgMember, err := isMultiOrgMember(cmd.Context(), *user)
+			if err != nil {
+				return err
+			}
+
+			if multiOrgMember && org == "" {
+				return xerrors.New("org is required for multi-org members")
+			}
+
+			client, err := newClient()
+			if err != nil {
+				return err
+			}
```

Comment:
Just noticed this. `isMultiOrgMember` should accept `client` as an argument. `newClient` will log a warning if the versions mismatch so we wouldn't want that to happen twice. It also makes a network call to see if the auth is valid which is wasteful to have twice.